lib/cdo/rake_utils.rb

```diff
@@ -160,7 +160,7 @@ def self.git_staged_changes?(path="")
   # full revision history needed to find the original commit SHA.
   def self.git_folder_hash(dir)
     Dir.chdir(File.expand_path(dir)) do
-      Digest::SHA2.hexdigest(`git ls-tree -r HEAD`)
+      Digest::SHA2.hexdigest(`git ls-tree -r HEAD 2>/dev/null`)
```

Comment:
Ignore stderr output to avoid polluting our test runs. (In the event of an error, the return value was and will continue to be the result of `Digest::SHA2.hexdigest('')`.